api/handler/gateway_action.go

```diff
@@ -918,6 +982,18 @@ func (g *GatewayAction) SyncHTTPRules(tx *gorm.DB, components []*apimodel.Compon
 	return db.GetManager().HTTPRuleDaoTransactions(tx).CreateOrUpdateHTTPRuleInBatch(httpRules)
 }
 
+func (g *GatewayAction) syncHTTPRuleRewrites(tx *gorm.DB, httpRules []*model.HTTPRule, rewrites []*model.HTTPRuleRewrite) error {
+	var ruleIDs []string
+	for _, hr := range httpRules {
+		ruleIDs = append(ruleIDs, hr.UUID)
```

Comment:
Should use `HTTPRuleID` ?

Reply:
Oh.. I forgot to implement the DeleteByHTTPRuleIDs method.

Reply:
It seems that you have implemented it, but the ID value here is wrong

Reply:
delete by single ruleID implemented, but not for multi ruldIDs...